app/scripts/templates/ready.mustache

```diff
@@ -1,14 +1,22 @@
 <div id="main-content" class="card">
   <header>
-    {{#signUp}}
-    <h1 id="fxa-sign-up-complete-header">{{#t}}Account verified{{/t}}</h1>
-    {{/signUp}}
-    {{#resetPassword}}
-    <h1 id="fxa-reset-password-complete-header">{{#t}}Password reset{{/t}}</h1>
-    {{/resetPassword}}
-    {{#accountUnlock}}
-    <h1 id="fxa-account-unlock-complete-header">{{#t}}Account unlocked{{/t}}</h1>
-    {{/accountUnlock}}
+    {{#isAccountUnlock}}
+      <h1 id="fxa-account-unlock-complete-header">{{#t}}Account unlocked{{/t}}</h1>
+    {{/isAccountUnlock}}
+    {{#isForceAuth}}
+      <h1 id="fxa-force-auth-complete-header">{{#t}}Welcome back{{/t}}</h1>
+    {{/isForceAuth}}
+    {{#isResetPassword}}
+      <h1 id="fxa-reset-password-complete-header">{{#t}}Password reset{{/t}}</h1>
+    {{/isResetPassword}}
+    {{#isSignIn}}
+      {{#isSync}}
+        <h1 id="fxa-sign-in-complete-header">{{#t}}Welcome to Sync{{/t}}</h1>
+      {{/isSync}}
+    {{/isSignIn}}
+    {{#isSignUp}}
+      <h1 id="fxa-sign-up-complete-header">{{#t}}Account verified{{/t}}</h1>
+    {{/isSignUp}}
```

Comment:
What I *really* want to do, and the initial approach I used, is to create distinct templates for all of these things, then choose the correct template depending on the `type` passed in to `initialize`. This resulted in 5 nearly identical templates. Template partials would reduce the boilerplate, but we haven't gotten that working with compiled Handlebars templates.